### ISSUE TITLE
Add a test script

### ISSUE THREAD

**Issue by marioortizmanero** [marioortizmanero/polybar-pulseaudio-control#14]
It doesn't have to be too detailed, but it'd be nice to have a test script to make sure the most basic functions always work as intended.

- [X] Test for `volUp`
- [X] Test for `volDown`
- [X] Test for `changeDevice`
- [ ] Test for `volSync`
- [x] Test for `volMute` (all three cases)
- [ ] `listen` behaviour (#9 and #20 should be finished first)
- [ ] More specific edge cases

**Comment by marioortizmanero** [comment]
I think there are enough tests for now.
